src/doc/rustdoc/src/unstable-features.md

```diff
@@ -106,7 +106,7 @@ item, it will be accompanied by a banner explaining that the item is only availa
 platforms.
 
 For Rustdoc to document an item, it needs to see it, regardless of what platform it's currently
-running on. To aid this, Rustdoc sets the flag `#[cfg(rustdoc)]` when running on your crate.
```

Comment:
This was probably the wrong line to remove. Looking at these docs again, they can probably stay as-is, if we rephrase the `cfg(rustdoc)` ones and link to them for more information.